src/localize/languages/it.json

```diff
@@ -2,83 +2,83 @@
   "$schema": "../schema/schema.json",
   "common": {
     "name": "Scheda Meteoalarm",
-    "description": "La scheda Meteoalarm ti informa degli eventi meteorologici in corso.",
-    "warning": null,
-    "expected": null,
+    "description": "La scheda Meteoalarm ti avvisa degli eventi meteorologici in corso.",
+    "warning": "Attenzione",
+    "expected": "Previsto",
     "unavailable": {
-      "long": null,
-      "short": null
+      "long": "L'integrazione non è disponibile",
+      "short": "Non disponibile"
     }
   },
   "error": {
     "missing_entity": "È necessario specificare un'entità!",
-    "invalid_integration": null,
+    "invalid_integration": "Questa integrazione non è valida!",
     "entity_invalid": {
-      "single": null,
-      "multiple": null
+      "single": "L'integrazione selezionata non corrisponde all'entità selezionata.",
+      "multiple": "L'integrazione selezionata non corrisponde alle entità selezionate: {entity}."
     }
   },
   "editor": {
-    "entity": null,
-    "integration": null,
-    "required": null,
-    "recommended": null,
-    "override_headline": null,
-    "hide_when_no_warning": null,
-    "hide_caption": null,
-    "disable_swiper": null,
+    "entity": "Entità",
+    "integration": "Integrazione",
+    "required": "Richiesto",
+    "recommended": "Consigliato",
+    "override_headline": "Sovrascrivere l'intestazione fornita dall'integrazione",
+    "hide_when_no_warning": "Nascondi quando non ci sono avvisi",
+    "hide_caption": "Nascondi la descrizione",
+    "disable_swiper": "Disabilità lo swipe delle allerte",
     "description": {
-      "start": null,
-      "current_expected": null,
-      "slots": null,
-      "separate_events": null,
-      "warning_watch_statement": null,
-      "end": null
+      "start": "L'integrazione selezionata crea più entità.",
+      "current_expected": "La prima con gli eventi attuali e la seconda con quelli futuri.",
+      "slots": "L'esatta quantità di è determinata dalla sua configurazione.",
```

Comment:
I find this translation unclear: maybe you mean "L'esatta quantità di **eventi** è determinata ..." ?

Reply:
In english is: "The exact amount of them created is determined by it's configuration", I thought it was part of a longer sentence

In inglese sembra essere parte di una frase più lunga, quindi l'ho lasciata con lo stesso significato

Reply:
(the review is outdated but is still valid because it is a context issue)

Reply:
![image](https://user-images.githubusercontent.com/26876994/186129178-39bbeeb9-3e9e-42c3-8beb-fb5c9a6c32f3.png)
@pmazz As you can see, it is part of a longer paragraph. So it is correct!